pulp/apis/scip_api.py

```diff
@@ -141,6 +149,20 @@ def actualSolve(self, lp):
         self.delete_tmp_files(tmpLp, tmpSol)
         return status
 
+    def getOptions(self):
+        params_eq = dict(
+            timeLimit="set limits time {}",
```

Comment:
one thing, could you take out the timeLimit here? in fact the timeLimit parameter is not stored in self.optionsDict. I know: it's anything but intuitive. Unfortunately, that's how it works with other CMD solvers too. If you check the `actualSolve` function, we assign the timeLimit from the `self.timeLimit` property. At least I want to keep all solvers "consistent".